src/Mod/Import/InitGui.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf8 -*-

```

Comment:
added `# -*- coding: utf8 -*-` to this and a few more files below to inhibit the unicode warning due to line 3